mkfig.sh

```diff
@@ -0,0 +1,11 @@
+if (( $# != 1 ))
+then
+  echo "Please supply figure name as first argument."
+  exit 1
+fi
+figname=$1
+curr_dir="$(dirname "$(realpath "$0")")"
```

Comment:
is this extracting the root directory where `mkfig.sh` is placed? Then copying this script in another directory in  `PATH` might screw things up. Would it work if it was symlinked instead? Then we should say that in the `README`